flink-runtime/src/main/java/org/apache/flink/streaming/api/operators/StreamSink.java

```diff
@@ -25,6 +25,7 @@
 import org.apache.flink.streaming.runtime.tasks.ProcessingTimeService;
 
 /** A {@link StreamOperator} for executing {@link SinkFunction SinkFunctions}. */
+@Deprecated
```

Comment:
With the addition of deprecation, it is usual to indicate in the comments that the method is deprecated (maybe in including the reason) what should be used instead.

Reply:
So do I. But the annotation exists in project all without comments.

Reply:
I am unsure why we are deprecating these methods. `SinkFunction`is not tagged as deprecated - some its methods are

Reply:
```
@Internal
@Deprecated
public interface SinkFunction<IN> extends Function, Serializable {
```